### DISCUSSION TITLE
question about solid stream example

### DISCUSSION THREAD

**Discussion by MadadiSaeed** [solidjs/solid#811] [Q&A] [answered]
In solid stream example rollup bundler outputs **almost** same components script for both client and server. why? What is the relationship between those client & server component? for example **Home** or **Profile** components .
[https://github.com/solidjs/solid/tree/main/packages/solid-ssr/examples/stream](https://github.com/solidjs/solid/tree/main/packages/solid-ssr/examples/stream)

**Comment by ryansolid** [comment, marked as answer]
Its what the DOM operations get compiled to mostly and a different runtime. On the server there is no reactivity and everything produces strings which gives optimal performance there. In the browser its direct DOM manipulation and reactivity to give the best performance there.

**Reply by MadadiSaeed** [reply]
Awesome. Thanks.
